solr/geodata-test/solrconfig.xml

```diff
@@ -182,7 +182,7 @@
          the PingRequestHandler.
          relative paths are resolved against the data dir
       -->
-    <str name="healthcheckFile">server-enabled.txt</str>
+    <!-- <str name="healthcheckFile">server-enabled.txt</str> -->
```

Comment:
Does this need to remain commented or should it be removed?

Reply:
That config requires the named file to exist in the data directory. If it's not there, then even though /admin/ping is configured it returns a 503. I figured this was the easiest way to get it working, the alternative being scaffolding that file into each core's datadir.

Note that Solr is firewall'd in staging / prod, so we're not opening things up any more than they currently are.

Reply:
👍 ; i think this is fine to go, then.